BentoKit/BentoKit/Views/FocusToolbar.swift

```diff
@@ -8,7 +8,9 @@ public final class FocusToolbar: UIToolbar {
     private var view: (UIView & FocusableView)?
 
     public init(view: UIView & FocusableView) {
-        let bundle = Bundle(for: FocusToolbar.self)
+        let url = Bundle(for: FocusToolbar.self).url(forResource: "BentoKit", withExtension: "bundle")
```

Comment:
So cocoa pods do not just add asset catalog to the framework target?

Reply:
I mean, why can't we just use `Bundle(for: FocusToolbar.self)`?

Reply:
If you use `resource_bundles` in the podspec it puts the resources into a separate bundle when making the pod version. I'll try with just `resources` to see if this avoids having one path for running locally and one for when part of the pod.